.github/workflows/upload_lambda_prod.yml

```diff
@@ -0,0 +1,59 @@
+name: Production deployment of Online DevOps Dojo coach lambda
+
+on:
+  push:
+    # change(s) on those files trigger this pipeline
+    paths: ['*.js', 'package*.json', 'serverless.yml', '.github/workflows/upload_lambda.yml']
+    branches: ['master']
+
+jobs:
+  build:
+    timeout-minutes: 10
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Copy the repository
+        uses: actions/checkout@v2
+      
+      # Node 12 included in LTS ubuntu-20.04 matches the version used for AWS Lambda
+      # (https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu2004-README.md)
+      # hence comment the upgrade code for now 
+      # - name: Check Node 12
+      #   uses: actions/setup-node@v1.4.2
+      #   with:
+      #     node-version: 12
+
+      # not supported yet for issue_comment event
+      # - name: Cache node modules  
+      #   uses: actions/cache@v2
+      #   env:
+      #     cache-name: cache-node-modules
+      #   with:
+      #     path: ~/.npm # npm cache files are stored in `~/.npm` on Linux/macOS
+      #     key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+      #     restore-keys: |
+      #       ${{ runner.os }}-build-${{ env.cache-name }}-
+      #       ${{ runner.os }}-build-
+      #       ${{ runner.os }}-
+
+      # Install nodejs packages listed in package-lock.json.
+      # - if: github.base_ref == 'master'
+      - name: Install node modules
+        run: npm ci --only=prod --no-fund
+
+      # - if: github.base_ref == 'master'
+      # run "serverless deploy --verbose" in a docker container and run the config file serverless.yml
+      - name: Deploy lambda
+        uses: serverless/github-action@v1.54.0
```

Comment:
I see v1.54 is out, however it doesn't look to be a FORMAL release yet?  is it ok?

![image](https://user-images.githubusercontent.com/33203301/87558399-1006ff00-c687-11ea-84a0-34fcf00a8d9e.png)

Reply:
This works fine but as you can see this is not maintained.
As I mentioned somewhere we could embed Probot in a GitHub Action instead of a AWS lambda. However I'm quite sure this would slowdown the bot reaction.
Else there is an [Ansible action](https://github.com/marketplace/actions/run-ansible-playbook) which is recent or a [CloudFormation action](https://github.com/marketplace/actions/aws-cloudformation-deploy-cloudformation-stack-action-for-github-actions) which also did not exist when I worked on this piece.

Regarding the _prod suffix of the the workflow file. **Good catch, this is a mistake.**
Actually I was on the way but never completed the pipeline to move in stage and prod with one workflow. Currently the workflow just deploy in the environment targeted by the credentials set in secrets. There is just one set of secrets.
I set the **test** environment secrets in my fork, which is not sustainable for multiple contributors.
However I was expecting more contributions on the training modules themselves than on the bot.